Multiple/Dual Frequency Oscillator/gerbers/BOM.md

```diff
@@ -0,0 +1,46 @@
+## BOM
+
+Here's a BOM (bill of materials) of all the parts you need for the different DFO boards. 
+Go to mouser.com and type in the part. no. Due to the pandemic and part shortage, many parts needed are still out of stock. ;(
```

Comment:
I would suggest too remove the second part of this line since it will be outdated.
In general try to avoid time-sensitive information since it will end up outdated.

Reply:
Good point. Link added to https://octopart.com/